packages/apps/intervention/src/hooks/use-intervention-header-actions.ts

```diff
@@ -120,3 +121,46 @@ const useInterventionDetailsActions = () => {
     mobileSettings?.isTrackerMessageEnabled,
   ]);
 };
+
+const useActiveInterventionActions = () => {
+  const I18n = useTranslator();
+  const Colors = useThemeColor();
+  const dispatch = useDispatch();
+
+  const {mobileSettings} = useSelector((state: any) => state.appConfig);
+  const {intervention, activeIntervention} = useSelector(
+    (state: any) => state.intervention_intervention,
+  );
+
+  useEffect(() => {
+    headerActionsProvider.registerModel('intervention_active_intervention', {
+      model: 'com.axelor.apps.intervention.db.Intervention',
+      modelId: intervention?.id,
+      disableMailMessages: !mobileSettings?.isTrackerMessageEnabled,
+      attachedFileScreenTitle: intervention?.sequence,
```

Comment:
mail messages, attached files and custom fields should be hidden too if intervention?.id !== activeIntervention?.id.
Maybe use activeIntervention here ?